### ISSUE TITLE
About Mouse Scrolling

### ISSUE THREAD

**Issue by Polawon** [OneLiberty/moonlight-chrome-tizen#34]
hi! Thanks to this project, I have Moonlight running on my Samsung TV. thank you!

I seem to be having some trouble using the mouse or trackpad. When I scroll with the mouse or trackpad, too many pages move at once in the browser or text viewer.

Is there a way to make the scrolling movement happen more slowly and smoothly?

**Comment by OneLiberty** [comment]
I think you can change this settings in sunshine. 
Did you check ?

**Comment by Polawon** [comment]
I check the configuration at the localhost sunshine but there's no options...(maybe I didn't know...). and if I configure some options at sunshine, isn't it affect when i use from other laptops??

**Comment by OneLiberty** [comment]
Didn't found one either, sorry for that. 
Did you try any other mouse ? 

Also does your mouse have a DPi setting or someting ? 

Cause' on my end using a MX master, i don't have any issue with scrolling speed :/

**Comment by JackTorrance1** [comment]
Hi, I also have the same problem and I can't fix it. This happens with any mouse and only on Samsung Moonlight client. There is an option in Sunshine: "High Resolution Scrolling Support
Default: checked" but it doesn't change anything.

**Comment by MrPhaze62** [comment]
If i recall correctly, some samsung TV has the menu to change scroll wheel speed within samsung tv settings, not sure if that may help.

**Comment by JohnnyPh** [comment]
> If i recall correctly, some samsung TV has the menu to change scroll wheel speed within samsung tv settings, not sure if that may help.

Yup it does fix the mouse move speed issue but not scroll speed though, thank you!
